x/pairing/keeper/unstaking.go

```diff
@@ -50,7 +60,6 @@ func (k Keeper) UnstakeEntry(ctx sdk.Context, chainID, creator, unstakeDescripti
 		"chainID":     existingEntry.GetChain(),
 		"geolocation": strconv.FormatInt(int64(existingEntry.GetGeolocation()), 10),
 		"moniker":     existingEntry.GetMoniker(),
-		"stake":       existingEntry.GetStake().Amount.String(),
```

Comment:
why just delete it out and not adding the amount that should've been unbonded?

Reply:
409afd20740ff51916b9fef473ca9b47f554f1ef